src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// dummy line 11
+// dummy line 12
+// dummy line 13
+// dummy line 14
+// dummy line 15
+// dummy line 16
+// violate HasTag rule: xoo on line 17
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35108/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:35108/project/issues?id=sample&branch=feature1&issues=AWD5cuR19JN_zULv01yL&open=AWD5cuR19JN_zULv01yL)

[ISSUE_KEY]: AWD5cuR19JN_zULv01yL

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED

Reply:
This issue has been reopened

[ISSUE_STATUS]: REOPENED